src/python/datastore/data_types.py

```diff
@@ -1294,6 +1294,12 @@ def _pre_put_hook(self):
                        coverage_information_key(self.fuzzer, self.date))
 
 
+class CorpusTag(Model):
+  """Corpus Tags for use in cross-pollination."""
+  tag = StringProperty()
+  fuzz_target = StringProperty()
```

Comment:
Nit: fuzz_target_name for consistency with other data types (like FuzzTargetJob).

Reply:
Done